birdhouse/config/canarie-api/docker_configuration.py.template

```diff
@@ -12,11 +16,77 @@ DATABASE = {
     'log_pid': '/var/run/nginx.pid'
 }
 
+
+def _get_release_time_from_github_tag(repository_reference, tagged_version):
+    # type: (str, str) -> str
+
+    repo = repository_reference.split("github.com/")[-1]
+    url = "https://api.github.com/repos/{}/tags".format(repo)
```

Comment:
`url = f'https://api.github.com/repos/{repo}/tags'`

This is f-string, since python 3.6, much more natural and readable.  Try to use this if you are 3.6 or higher for all your strings.